app/helpers/date-format.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from '@ember/component/helper';
+import { format } from 'date-fns';
```

Comment:
I don't know what you mean. Can we bundle only a function instead of the whole package? Do you mean something similar as the `lodash.somefunction` packages? I don't know if date-fns supports that.

I tried something like `import { format } from 'date-fns/format';` but that doesn't work.

Reply:
Yea, Webpack can treeshake unused functions, but it can't for some reason if you import from the top level package.

I think it's a default export so like this: `import format from 'date-fns/format'`.

Reply:
You can check quickly by doing a production build and comparing the size changes. Maybe the problem has been fixed and the subpath import isn't needed anymore.

Reply:
For reference, this is the underlying issue that prevents it from working without the subpath: https://github.com/ef4/ember-auto-import/issues/121

So until that is fixed, or we switch to Embroider, the subpath is needed to reduce the bundle size.

Reply:
The package size difference between the two build (one with subpath and one without) is 75.632 bytes. So I will use the subpath method.